### PR TITLE
CAL-295 added content to transformers docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Documents Alliance transformers.

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth @bdeining @troymohl 

#### Choose 2 committers to review/merge the PR.
@jaymcnallie
@jlcsmith
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested?

review new content.

#### What are the relevant tickets?

[CAL-295](https://codice.atlassian.net/browse/CAL-295)

#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_transformers/chipping-metacard-transformer-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+:title: Chipping Metacard Transformer
+:type: transformer
+:subtype: input
+:status: published
+:link: _chipping_metacard_transformer
+:summary: Extracts chips from NITF images.
+
+The Chipping Metacard Transformer extracts chips from NITF images.
```
Comment (bdeining) [inline distribution/docs/src/main/jdocs/content/_transformers/chipping-metacard-transformer-contents.adoc]: The ImagingChipTransformer also supports cropping

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_transformers/chipping-metacard-transformer-contents.adoc]: will add.

File: distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.orb.impl.corbaorb-table-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+|${org.codice.alliance.corba_default_port}
+|true
+
+| CORBA Timeout
+| corbaTimeout
+| Integer
+| The timeout (seconds) associated with CORBA reads.
+| 60
+| true
+
```
Comment (troymohl) [inline distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.orb.impl.corbaorb-table-contents.adoc]: Are these spaces before the text intentional? I've noticed some of the table entries have them and others don't

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.orb.impl.corbaorb-table-contents.adoc]: Nope. Will fix.

File: distribution/docs/src/main/jdocs/content/_tables/NSILI_Connected_Source-table-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,225 @@
+:type: documentation
+:status: published
+
+.[[NSILI_Federated_Source]]NSILI Federated Source
```
Comment (mcalcote) [inline distribution/docs/src/main/jdocs/content/_tables/NSILI_Connected_Source-table-contents.adoc]: Should this be NSILI Connected Source?

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_tables/NSILI_Connected_Source-table-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**